File: PUSGS/src/app/Entities/work-request.ts
import { WorkRequestBasicInfo } from "./work-request-basic-info";
import { WorkRequestEquipments } from "./work-request-equipments";
import { WorkRequestHistoryOfChanges } from "./work-request-history-of-changes";
import { WorkRequestMultimedia } from "./work-request-multimedia";

export class WorkRequest {
    basicinfo: WorkRequestBasicInfo;
    historyofchanges: Array<WorkRequestHistoryOfChanges>;
    multimedia: Array<WorkRequestMultimedia>;
    equipments: Array<WorkRequestEquipments>;

    constructor(){}
}

File: PUSGSVeb2/PUSGSVeb2/Models/WorkRequest.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace PUSGSVeb2.Models
{
    public class WorkRequest
    {
        [Key]
        public int id { get; set; }

        public WorkRequestBasicInfo basicinfo { get; set; }

        public ICollection<WorkRequestHistoryOfChanges> historyofchanges { get; set; }

        public ICollection<WorkRequestMultimedia> multimedia { get; set; }

        public ICollection<WorkRequestEquipments> equipments { get; set; }
    }
}
